Sources/ComposableArchitecture/Store.swift

```diff
@@ -455,30 +448,59 @@ public final class Store<State, Action> {
       Action
   ) -> Store<PresentationState<ChildState>, PresentationAction<ChildAction>> {
     self.scope(
-      state: toChildState,
+      state: ToState(toChildState),
       id: nil,
       action: fromChildAction,
       isInvalid: nil,
       removeDuplicates: { $0.sharesStorage(with: $1) }
     )
   }
 
-  @_spi(Internals) public func scope<ChildState, ChildAction>(
-    state toChildState: @escaping (State) -> ChildState,
-    id: ScopeID<State, Action>?,
-    action fromChildAction: @escaping (ChildAction) -> Action,
-    isInvalid: ((State) -> Bool)?,
-    removeDuplicates isDuplicate: ((ChildState, ChildState) -> Bool)?
```

Comment:
- [ ] We only supply an `isDuplicate` argument when scoping to presentation state/actions, but in this PR the argument is now effectively ignored. Before merging, we should figure out if it is really doing much and if we want to continue to support it. If we don't want to continue to support it we should remove it entirely from the internal APIs.